tradewave/templates/tradewave/vendor-assign-users.html

```diff
@@ -7,14 +7,15 @@
 {% block content %}
 
   <!-- User Form -->
-  <form method="post" action="{% url 'tradewave:entity_assign_users' entity_id=vendor_id %}">
+  <form method="post" action="{% url 'tradewave:entity_assign_users' entity_id=53 %}">
```

Comment:
@dannypernik Still have not figured out how to pass the vendor id here, so this is hardcoded for the time being.